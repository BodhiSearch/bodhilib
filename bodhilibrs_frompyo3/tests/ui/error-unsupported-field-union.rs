use frompyo3::FromPyO3;

#[derive(FromPyO3)]
union TestUnsupportedUnion {
  weight: i32,
  temp: f32,
}

fn main() {}
