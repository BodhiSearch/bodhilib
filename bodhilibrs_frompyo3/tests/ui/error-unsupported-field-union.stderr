error: Unions are not supported
 --> tests/ui/error-unsupported-field-union.rs:4:1
  |
4 | / union TestUnsupportedUnion {
5 | |   weight: i32,
6 | |   temp: f32,
7 | | }
  | |_^
